dist/profile/manifests/buildmaster.pp

```diff
@@ -233,9 +234,16 @@
     enabled =>        false, # Disabled by default to avoid messing up with unmanaged instances
     custom_configs => [],
     reload_token =>   '',
-    common_configs => ['buildmaster/casc/clouds.yaml.erb'],
+    common_configs => [ # Configurations shared by all Jenkins controllers. There are hieradata attribute to really fill the configs (as an additional step for opt-in)
+      'buildmaster/casc/clouds.yaml.erb',
+      'buildmaster/casc/tools.yaml.erb',
+    ],
     config_dir =>     'casc.d', # Relative to the jenkins_home
   }
+
+  ### Deep merge the different sources (becasue hieradata 3 failsa t deep merge, but with hieradata 5 we will be able to delegate)
```

Comment:
```suggestion
  ### Deep merge the different sources (because hieradata 3 failsa t deep merge, but with hieradata 5 we will be able to delegate)
```